### DISCUSSION TITLE
Adding Additional Buttons to the modal

### DISCUSSION THREAD

**Discussion by cryptobredda** [WalletConnect/walletconnect-monorepo#2672] [Developer Support] [answered]
Hi,

I would like to add additional buttons to the modal so such as Referal Link, Profile, etc...

I am wondering if I can do that easily with this or do I need to create my own Account modal component?

I know that I can use each of the 4 models seperately so I know I can use the default connet modal but use a custom Account modal if needed.

I am just wondering if there is an easy way instead of me spending time making a custom one xD

**Comment by xzilja** [comment, marked as answer]
Nothing like this available at the moment, we mostly allow you to theme the modal / choose wallets / chains etc. but for more custom solution like what you described, a custom ui will be needed.
